app/views/job_invocations/_form.html.erb

```diff
@@ -81,5 +84,20 @@
     </fieldset>
   </div>
 
+  <!-- modal window -->
```

Comment:
I don't think we'd have to migrate all modal windows in core but I though this could be a good start which could be extended when needed as we might need more modal windows in rex... but I don't insist, I just wanted to have the form partial clean, so at least partial would be good